DataFiles/reference-documents.csv

```diff
@@ -536,10 +536,12 @@ EN ISO 8394-1:2010,2010-10-01,Building construction - Jointing products - Part 1
 EN ISO 8394-2:2017,2017-11-22,Buildings and civil engineering works - Determination of extrudability for sealant - Part 2: Using standardized apparatus (ISO 8394-2:2017)
 EN ISO 9001,1900-01-01,quality management systems - requirements
 EN ISO 9053-1:2018,2018-12-12,Acoustics - Determination of airflow resistance - Part 1: Static airflow method (ISO 9053-1:2018)
+European Illustrated Glossary of Conservation Terms for Wall Paintings and Architectural Surfaces,2015-01-01,EwaGlos – European Illustrated Glossary of Conservation Terms for Wall Paintings and Architectural Surfaces
 FHWA-NHI-23-024,2023-03-01,Bridge Inspector’s Reference Manual (BIRM) (2022 NBIS)
 IEC 60050-351,2014-09-01,International electrotechnical vocabulary - Part 351: Control technology
 IEC 61987-1,2006-12-01,Industrial-process measurement and control – Data structures and elements in process equipment catalogues – Part 1: Measuring equipment with analogue and digital output
 IEC CDD,1980-01-01,IEC Common Data Dictionary
+Illustrated Glossary on Stone Deterioration Patterns,2008-09-01,Illustrated Glossary on Stone Deterioration Patterns
```

Comment:
Same here, the code is not a code and it is too long (max 50 chars).
Suggestion: IGSDP-2008